Reqnroll.VisualStudio/ProjectSystem/Settings/ReqnrollProjectSettingsProvider.cs

```diff
@@ -210,13 +210,27 @@ private NuGetPackageReference GetSpecFlowPackage(IProjectScope projectScope,
 
     private string GetReqnrollConfigFilePath(IProjectScope projectScope)
     {
-        var projectFolder = projectScope.ProjectFolder;
         var fileSystem = projectScope.IdeScope.FileSystem;
-        return fileSystem.GetFilePathIfExists(Path.Combine(projectFolder,
-                   ProjectScopeDeveroomConfigurationProvider.ReqnrollJsonConfigFileName)) ??
-               fileSystem.GetFilePathIfExists(Path.Combine(projectFolder,
-                   ProjectScopeDeveroomConfigurationProvider.SpecFlowJsonConfigFileName)) ??
-               fileSystem.GetFilePathIfExists(Path.Combine(projectFolder,
-                   ProjectScopeDeveroomConfigurationProvider.SpecFlowAppConfigFileName));
+        var assemblyDirectory = Path.GetDirectoryName(projectScope.OutputAssemblyPath);
```

Comment:
I think there are cases when `projectScope.OutputAssemblyPath` is null (when the project was never built?), so I would feel more safe if we would calculate this if the `projectScope.OutputAssemblyPath` is not null. Or do we use this property in the other parts of the code without checking for null? Could you please check?

Reply:
Path.GetDirectoryName(projectScope.OutputAssemblyPath) will return null if projectScope.OutputAssemblyPath is null.
https://learn.microsoft.com/en-us/dotnet/api/system.io.path.getdirectoryname?view=net-8.0

> Returns
[String](https://learn.microsoft.com/en-us/dotnet/api/system.string?view=net-8.0)
Directory information for path, or null if path denotes a root directory or is null. Returns [Empty](https://learn.microsoft.com/en-us/dotnet/api/system.string.empty?view=net-8.0#system-string-empty) if path does not contain directory information.

That's why the `if (assemblyDirectory != null)` check is there

Reply:
Perfect. Thx. (I did not remember or check how the `Path.GetDirectoryName` behaves for nulls... 😎